py/setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import setuptools
+import shutil
+
+# Copies the schema in the module so that setuptools is able to
+# find the file and add it to the package.
+if os.path.isfile("../relay/event.schema.json"):
+    shutil.copyfile(
+        "../relay/event.schema.json",
+        "./sentry_data_schemas/event.schema.json"
+    )
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="sentry-data-schemas",
+    version="0.0.1",
+    author="Sentry",
+    license="BSL-1.1",
+    author_email="hello@sentry.io",
+    description="Sentry shared data schemas",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/getsentry/sentry-data-schemas",
+    packages=setuptools.find_packages(),
+    include_package_data=True,
+    install_requires=[
+        "jsonschema-typed-v2==0.8.0"
+    ],
+    python_requires='>=3.8',
```

Comment:
I'm not going to be consuming this, but this seems pretty restrictive if we wanna use this elsewhere considering we hardly depend on python 3.8 (only in snuba), and our SDKs would never enforce this or our server for a while.

Reply:
I think I can go down to 3.6 or something around that by importing TypeDict from mypy.experimental as we were doing before 3.8.